core/primitives/src/views.rs

```diff
@@ -1075,7 +1112,32 @@ impl From<FinalExecutionOutcomeWithReceiptView> for FinalExecutionOutcomeView {
 }
 
 #[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Debug, Clone, Eq, PartialEq)]
-pub struct ValidatorStakeView {
+pub enum ValidatorStakeView {
```

Comment:
@frol I don't know if updating `ValidatorStakeView` to be an enum has any backwards compatibility implications for json rpc. If so, let me know and we can discuss how to resolve.